fabfile.py

```diff
@@ -41,6 +43,7 @@
 
 
 DEFAULT_FILTERS_FILES = 'configs/filters'
+ERROR_INVALID_CONFIGURATION = 2
```

Comment:
Duplicate definition of `ERROR_INVALID_CONFIGURATION`, please use the one defined in previously.